frontend/src/components/TransactionRows/AmountAndName.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="amount-and-name">
+    <b-row>
+      <b-col cols="5">
+        <div class="text-right">
+          <span class="gdd-transaction-list-item-amount">
+            {{ amount | GDD }}
+          </span>
+        </div>
+      </b-col>
+      <b-col cols="7">
+        <div class="gdd-transaction-list-item-name">
+          {{ itemText }}
+        </div>
+      </b-col>
+    </b-row>
+  </div>
+</template>
+<script>
+export default {
+  name: 'Amount-And-Name',
```

Comment:
Wollen wir das nicht Row nennen?

Reply:
umbenannt in AmountAndNameRow, amunt-and-name-row